tests/mmt/model.mmt

```diff
@@ -0,0 +1,90 @@
+[[model]]
```

Comment:
Could you please add a short description what this model is about or why this model is used as a test.